app.py

```diff
@@ -25,6 +27,37 @@ def setup(self):
         context().full_importer = FullImport()
         context().inc_importer = IncrementalImport()
 
+    def get_schema_extension(self):
+        return SchemaExtension(
+                key = '1f6ab337-b1ac-4558-8dc5-e97b4c666415',
+                owner = 'Tenable IO Connector',
+                version = '1',
+                schema = '''
+                {
+                    "vertices": [
+                        {
+                            "name": "vulnerability",
+                            "properties": {
+                                "links": {
```

Comment:
This may need to be an array of strings, similar to: https://github.com/IBM/cp4s-car-schema/blob/master/doc/generated/importSchema_v2.json#L44

Reply:
jsonb works well